### PR TITLE
[touchtts] fix bug and support chunk-mask for touch_flow

### PR DESCRIPTION
**Pull request by chwma0**

### CODE REVIEW

File: west/models/touch_flow/modeling_touch_flow.py [hunk 1]
```diff
@@ -212,8 +223,18 @@ def inference(
         x_in[0:1, :, M:2 * M] = token_cond
         x_in[0:1, :, 3 * M:4 * M] = spk_cond
         x_in[0:1, :, 4 * M:5 * M] = mel_cond
-        vocoder_lengths = torch.tensor([T], dtype=torch.long, device=device)
-        att_mask = non_causal_mask(vocoder_lengths).to(device)  # (B, T, T)
+        mask = ~make_pad_mask(mel_vocoder_lengths).to(device)  # (B, T)
+        att_mask = add_optional_chunk_mask(
```
Comment (robin1001) [inline west/models/touch_flow/modeling_touch_flow.py:228]: 这个函数本身是在 wenet 中给识别设计的，因历史原因，越来越长了，可以看看怎么精简一下，仅保留必要的部分

Comment (robin1001) [reply west/models/touch_flow/modeling_touch_flow.py:228]: Cosyvoice2 中的策略是这样

image

Comment (chwma0) [reply west/models/touch_flow/modeling_touch_flow.py:228]: modify comments

**Review by robin1001 — APPROVED** [review]
